libs/dh/esett/data-access-outgoing-messages/src/index.ts

```diff
@@ -17,5 +17,4 @@
  */
 //#endregion
 export { DhEsettDataAccessApiStore } from './lib/dh-esett-data-access-api.store';
-export { DhOutgoingMessagesSignalStore } from './lib/dh-outgoing-messages.store';
```

Comment:
nitpick: the actual file can be deleted as well

Reply:
You are right, will remove in followup PR